scripts/sv/callVariants.sh

```diff
@@ -21,12 +21,11 @@ echo "Assuming 2-bit reference: " "$TWOBIT_REFERENCE_LOCATION"
 
 cd "$GATK_DIR" 
 
-./gatk-launch CallVariantsFromAlignedContigsSpark \
-    --inputAlignments "$PROJECT_OUTPUT_DIR"/aligned_assemblies \
-    --inputAssemblies "$PROJECT_OUTPUT_DIR"/assembly_0 \
+./gatk-launch CallVariantsFromAlignedContigsSAMSpark \
```

Comment:
Ah.  So the right action is to put SAM parsing into CallVariantsFromAlignedContigsSpark?
I can probably do that.  So far I've been careful not to disrupt the old SGA pipeline, but this would break that.  Might it be better for one of us to move your indel parsing code into ...SAMSpark?